docs/configuration/sendrecvoptions.rst

```diff
@@ -155,5 +178,29 @@ If they are installed on the receiving side as well as the sending side, propert
 Below is an incomplete list of problematic properties.
 If you have another entry for this list, please open an issue or pull request for this page.
 
+Mount behaviour
+---------------
+
 * ``mountpoint``
 * ``canmount``
+* ``overlay``
+
+Encryption
+----------
+
+If the source datasets are encrypted but you do **not** set ``encrypted`` or ``raw`` in the :ref:`send options<job-send-options>` to ``true`` on the sending side (i.e. you do an unencrypted send) **and** you enable property replication, you will have to inherit the following properties on the :ref:`receiving side<job-recv-options>`:
```

Comment:
Don't use `you` here.

Reply:
Also, maybe explain why one would want to do this? (I suppose: receiving an unencrypted stream into an encrypted pool that has keys loaded?)

Reply:
Site a->Site b
Site a is encrypted. You'll either have to send it encrypted or plain, so for distribution reasons you opt for plain.
You do want to replicate properties though for backup completeness.

This I would suggest making a tutorial section for, that goes through most of the permutations of replication sides being encrypted, `{raw,encrypted}={true,false}`, properties on/off, etc.

Tl;dr: IMHO keep this mostly about the specific zrepl feature and the underlying zfs mechanisms and reserve the tutorials section for bigger setup and use case instructions

Reply:
OK, but still the `you` should go away *and* the sentence is way too long / complex / has too many conditionals.